### PR TITLE
add a separate docker-compose file for development

### PR DESCRIPTION
**Pull request by robojones**
Part of Gamify-IT/issues#82

### CODE REVIEW

**Review by delvh — DISMISSED** [review]
Seems fine to me.

File: nginx/dev.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header   X-Forwarded-Host $server_name;
+
+    location / {
+        proxy_pass      http://login-frontend:80/;
+    }
+
+    location /api/login/ {
+        proxy_pass      http://host.docker.internal:80/;
+    }
```
Comment (madebyTimo) [inline nginx/dev.conf:17]: sure this works?

If I see it right you proxy the api/login/ to thee host machine. Port 80 at host machine then is forwarded to the nginx and there it is proxied the frontend.

**Review by madebyTimo — CHANGES_REQUESTED** [review]

**Review by LeviUniAcc — APPROVED** [review]

File: nginx/dev.conf (hunk 1 above)
Comment (robojones) [reply nginx/dev.conf]: Good point. Not sure what I was thinking there

**Review by delvh — APPROVED** [review]

**Review by madebyTimo — APPROVED** [review]
Looks fine, but why you made a force push? We should avoid force pushed, see point 5 [here](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/repositories/repository-guide.html?#pull-requests)

**Pull request merged**